CMakeLists.txt

```diff
@@ -0,0 +1,261 @@
+cmake_minimum_required(VERSION 3.6)
+
+project(breakpad VERSION 1.0)
```

Comment:
Since there is no upstream releases yet I think it's better to start with `0.0.0` to avoid conflicts or don't specify VERSION at all.